syntax="proto3";
package echo;

option py_generic_services = true;
option cc_generic_services = true;

message RequestMessage
{
	int32 id = 1;
	string msg = 2;
}

message ResponseMessage
{
	int32 id = 1;
	string msg = 2;
}

//rpc service
service EchoService
{
	rpc echo(RequestMessage) returns(ResponseMessage);
}